### ISSUE TITLE
TON_FG-HR

### ISSUE THREAD

**Issue by ektrah** [qudt/qudt-public-repo#678]
We are mapping BACnet units to QUDT units, and are currently looking at the following two items:

```
tons-refrigeration                                    (Power Unit)
ton-hours                                             (Energy Unit)
```

In QUDT we find `unit:TON_FG`, which corresponds to tons-refrigeration. However, there doesn't seem to be a `qudt:TON_FG-HR`.

Is there perhaps already an official mapping somewhere that we could use, or would it be possible to add `qudt:TON_FG-HR` to QUDT?

**Comment by steveraysteveray** [comment]
Indeed, we would invite such a submission. Please take a look at our [Units submission guidelines](https://github.com/qudt/qudt-public-repo/wiki/Unit-Vocabulary-Submission-Guidelines).

**Comment by steveraysteveray** [comment]
Thanks! Fixed in PR #692.

**Comment by ektrah** [comment]
Thank you very much!

**Comment by steveraysteveray** [comment]
Thanks for your support! $$$
